google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudType.java

```diff
@@ -161,11 +160,27 @@ public ManagedVmServerConnector(
 
     @Override
     public void connect(@NotNull ConnectionCallback<ManagedVmDeploymentConfiguration> callback) {
-      callback.connected(new ManagedVmRuntimeInstance(configuration));
+      try {
+        getFileFromFilePath(configuration.getCloudSdkExecutablePath());
+        callback.connected(new ManagedVmRuntimeInstance(configuration));
+      } catch(RuntimeException re) {
+        callback.errorOccurred("Invalid Cloud SDK directory path configured.");
+        // TODO Consider auto opening configuration
+      }
+    }
+  }
+
+  @NotNull
+  private File getFileFromFilePath(String filePath) {
+    File file;
+    file = new File(filePath);
+    if (!file.exists()) {
+      throw new RuntimeException(filePath + " does not exist");
     }
+    return file;
   }
 
-  private static class ManagedVmRuntimeInstance extends
+  private class ManagedVmRuntimeInstance extends
```

Comment:
Shouldn't rather the `getFileFromFilePath(String)` become static? It's not accessing the internal state of `ManagedVmCloudType` and neither does `ManagedVmRuntimeInstance`, so making them static seems better.